app/layout.tsx

```diff
@@ -18,7 +18,7 @@ export default function RootLayout({
 }>) {
   return (
     <html lang="en">
-      <body className={inter.className}>
+      <body className={inter.className + ' p-6'}>
```

Comment:
Shall we use [classnames](https://www.npmjs.com/package/classnames)?

Reply:
Resolved in [7a0879d](https://github.com/imperial/ug-admissions/pull/3/commits/7a0879d032fd72411c6f4a8358a4bd6cae18dc33)